server/build.gradle

```diff
@@ -123,6 +123,10 @@ dependencies {
   // repackaged jna with native bits linked against all elastic supported platforms
   compile "org.elasticsearch:jna:${versions.jna}"
 
+  compile "com.fasterxml.jackson.core:jackson-databind:${versions.jackson}"
```

Comment:
I have abandoned the idea of using  Json layout for the moment. Not needed for current hardcoded pattern layout

Reply:
I'm quite weary of having this jar with the server's permissions. I *think* it is the one that usually sees security issues when Jackson gets security problems.

Reply:
Yes - I haven't decided to use it Json layout after all, will remove related code from the review. It was good to confirm that it is not what we want to follow, thanks for looking into this bit